.github/workflows/lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint
+
+on:
+  push:
+    paths:
+      - '*.py'
+      - '!**.md'   
```

Comment:
Do we need this declaration (is `*.py` enough)?

Reply:
It doesn't need to run if only ,md files are changed

Reply:
doesn't the `*.py` path mean "only when this path/pattern is matched"?

Reply:
It does (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths). It'll only run when there are changes to a .py file. 

Although I can see as the build and push are dependent on this job running you would want it to run in more conditions, only not running which changes are to **.md.

Reply:
Fixed in https://github.com/geopython/pygeoapi/pull/1002